.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Check
+
+on:
+  push:
+    branches:
+      - master
+      - releases/v[0-9]+.[0-9]+.[0-9]+
+  pull_request:
+    types: [opened, synchronize]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  validate_copyright:
+    name: Copyright
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        show-progress: false
+    - name: Install dependencies
+      run: python3 -m pip install --user python-debian
+    - name: Validate copyright file with python-debian
+      run: python3 ./utils/check_copyright.py
```

Comment:
Doesn't exist in this repository (should it be added)?
```suggestion
```

Reply:
In my brain it made sense to have the same checks here as in the main repo. I do think the old copyright check is redundant now, as the new one seems to be working reliably. So yeah, maybe adding this here isn't too useful.